testsuite/docs-examples-python/src/docs-examples-imagebufalgo.py

```diff
@@ -263,10 +263,10 @@ def example_make_texture():
     config["maketx:filtername"] = "lanczos3"
     config["maketx:opaque_detect"] = 1
 
-    ok = ImageBufAlgo.make_texture (OpenImageIO.MakeTxTexture,
+    ok = ImageBufAlgo.make_texture (oiio.MakeTxTexture,
```

Comment:
For consistency with other examples, should this use `MakeTxTexture` instead of `oiio.MakeTxTexture`? All the other examples use stuff imported from `from OpenImageIO import *`.

Reply:
Sure.

I'm not very immersed in Python culture. Remembering that this is not just a test, but its snippets will be directly used in the documentation as examples for the API calls, what's customary when documenting a library and providing code examples? Is it typical to omit any namespacing at all? Or to use a namespace in the examples? To always use the full namespace?

Reply:
Numpy example: https://numpy.org/doc/1.26/user/absolute_beginners.html#how-to-import-numpy

They show how to import, use a short namespace alias, and then they use the namespace everywhere in the rest of the examples. I think this is wise, as for beginners it more easily distinguishes between the Numpy API and things that might be features of Python itself.

I think we should do the same. So I want to keep the use of the oiio namespace here. And let's make a mental note that, separately, we should find anyplace in the examples that does a full `from OpenImageIO import *` and change that to use a namespace to be more clear.

Reply:
If possible, it's recommended to avoid using start imports.

Reply:
Agreed. In general, in both C++ and Python, I think it's better style to always use the namespacing explicitly, for clarity. At least as a default, and take exceptions one by one in cases where removing the namespacing clearly the code more readable.

Reply:
I'm not sure where the `*` import came from. Git blame says me, of course, but it's contrary to my usual style, so I think I just wasn't considering it very carefully at that moment.